core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -1054,7 +1054,8 @@ class KafkaConfigTest {
   private def assertValidQuorumVoters(value: String, expectedVoters: util.Map[Integer, AddressSpec]): Unit = {
     val props = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect)
     props.put(RaftConfig.QUORUM_VOTERS_CONFIG, value)
-    val raftConfig = new RaftConfig(KafkaConfig.fromProps(props))
+    val kafkaConfig = KafkaConfig.fromProps(props)
+    val raftConfig = new RaftConfig(kafkaConfig, kafkaConfig.replicaFetchResponseMaxBytes)
```

Comment:
Why should we put an additional parameters into `RaftConfig` here? The `replicaFetchResponseMaxBytes` is already in `kafkaConfig`, isn't it?

Reply:
make sense.